submissions/zklash.md

```diff
@@ -0,0 +1,39 @@
+# Game Jam Submission Template
+
+# zKlash
+
+### Project Summary
+Give a short summary of your game project.
+
+Autobattler where you buy creatures and improve them to defeat waves of enemies
+
+### GitHub
+Provide the GitHub where your project is hosted. Please ensure your github repo is registered with [OnlyDust](https://app.onlydust.com/p/create) to receive awards.
+
+https://github.com/z-korp/zklash [OnlyDust](https://app.onlydust.com/p/zklash).
```

Comment:
Awesome! I would suggest to change the project to Z-KORP and have zklash as a repo inside of it, it follows the github org structure, then awards can be allocated to Z-KORP